### PR TITLE
[#166818022] Added component for empty local services

### PR DESCRIPTION
**Pull request by laurapugliese**
Added component for empty local services

### CODE REVIEW

**Comment by digitalcitizenship** [comment]
## Affected stories

  * 🌟 [#166818022](https://www.pivotaltracker.com/story/show/166818022): Come CIT nel tab servizi "locali" voglio poter vedere il pulsante  "Aggiungi le tue aree di interesse"

<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

**Comment by laurapugliese** [comment]
In this PR for compatibility with small device (ex. iPhone SE) I added the animation for hide the header of screen on scroll in all tabs in the same way as the message section (see gif below).
@fpersico @Undermaken an improvement could be to outsource this logic to the tabs in a new component or integrate it into the existing AnimatedScreenContent component (and use it in Messages section and new Services section). If you agree we can create a new story for this. I await your feedback.

aggiungi_aree_interesse

**On iPhone SE:**
aggiungi_aree

File: ts/components/services/OtherServices.tsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  paddingForAnimation: boolean;
+  AnimatedCTAStyle?: any;
+};
+
+type Props = AnimatedProps;
+
+/**
+ * A component to render a list of other services.
+ */
+class OtherServices extends React.PureComponent<Props> {
```
Comment (fpersico) [inline ts/components/services/OtherServices.tsx]:
```suggestion
class ServicesOther extends React.PureComponent<Props> {
```

The other component is called ServicesLocal.

Comment (laurapugliese) [reply ts/components/services/OtherServices.tsx]: @fpersico done in commit 17dc7ae

File: ts/screens/services/ServicesHomeScreen.tsx [hunk 2]
```diff
@@ -127,7 +151,47 @@ class ServicesHomeScreen extends React.Component<Props, State> {
               </Text>
             </TabHeading>
           }
-        />
+        >
+          <ServicesLocal
+            animated={{
+              onScroll: Animated.event(
```
Comment (Undermaken) [inline ts/screens/services/ServicesHomeScreen.tsx]: Should **onScroll handler** be defined one time and returned from a given index?

File: ts/screens/services/ServicesHomeScreen.tsx [hunk 3]
```diff
@@ -136,7 +200,47 @@ class ServicesHomeScreen extends React.Component<Props, State> {
...
+                      }
+                    }
+                  }
+                ],
+                { useNativeDriver: true }
+              ),
+              scrollEventThrottle: 8 // target is 120fps
+            }}
+            paddingForAnimation={true}
+            AnimatedCTAStyle={{
```
Comment (Undermaken) [inline ts/screens/services/ServicesHomeScreen.tsx]: Should **transform** object be defined one time and called within state currentTab?

**Pull request merged**